docs/api/service-index.md

```diff
@@ -65,6 +65,10 @@ There is no requirement that each resource has a unique `@id` or `@type`. It is
 determine which resource to prefer over another. One possible implementation is that resources of the same or
 compatible `@type` can be used in a round-robin fashion in case of connection failure or server error.
 
+A resource can use a different host or domain than the service index, but this may cause issues in environments with strict network rules.
+If you delegate resources to api.nuget.org, your feed will not work where access to nuget.org is blocked.
```

Comment:
```suggestion
If you delegate resources to nuget.org, your feed will not work where access to nuget.org is blocked.
```

Technically our service index points to several other nuget.org domains (search, gallery, symbols). Maybe this can be generalized to this 👆? Not sure.